packages/design-system-react/src/components/AvatarGroup/AvatarGroup.stories.tsx

```diff
@@ -140,10 +140,7 @@ export const Size: Story = {
   render: () => (
```

Comment:
Moved to a separate PR https://github.com/MetaMask/metamask-design-system/pull/683